SpareMe/screens/CreateAccount.js

```diff
@@ -0,0 +1,179 @@
+'use strict';
+import React, { Component } from 'react';
+import { Alert, StyleSheet, Text, View, Button, TextInput, NetInfo } from 'react-native';
+import CustomStatusBar from '../components/CustomStatusBar'
+import FilterWebView from '../components/FilterWebView'
+import firebase from 'react-native-firebase';
+import * as constants from 'constants'
+
+export default class CreateAccount extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {};
+        NetInfo.isConnected.fetch().then(isConnected => {
+            console.log(isConnected);
+            this.setState({isConnected: isConnected});
+        });
+    }
+
+    componentDidMount() {
+        NetInfo.isConnected.addEventListener('connectionChange', this.onConnectivityChange);
+    }
+
+    componentWillUnmount() {
+        NetInfo.removeEventListener('connectionChange', this.onConnectivityChange);
+    }
+
+    onConnectivityChange = isConnected => {
+        this.setState({isConnected: isConnected});
+    }
+
+    onRegister = () => {
+        const { email, password } = this.state;
+        firebase.auth().createUserWithEmailAndPassword(email, password)
+        .then((user) => {
+          this.props.navigation.goBack();
+            // If you need to do anything with the user, do it here
+            // The user will be logged in automatically by the
+            // `onAuthStateChanged` listener we set up in App.js earlier
+        }).catch((error) => {
+            const { code, message } = error;
```

Comment:
Does `message` give a reason for failure? It'd be cool if we could say "username already taken", "password doesn't meet requirements", etc